requirements.in

```diff
@@ -16,3 +16,6 @@ pyjwt
 requests-futures
 # we require f8a_worker, but it is installed manually as it is not available in PyPI
 #f8a_worker
+
+google-cloud
+joblib
```

Comment:
The same as in https://github.com/fabric8-analytics/fabric8-analytics-jobs/pull/100#discussion_r127678657. It should probably go to worker unless these deps are not directly used by server.